functions/Get-DbaAgentJob.ps1

```diff
@@ -67,18 +77,26 @@ FUNCTION Get-DbaAgentJob {
 			}
 
 			$jobs = $server.JobServer.Jobs
+			if ($Enabled) {
+				if($Enabled -eq 'True') { 
+					$jobs = $Jobs | Where-Object IsEnabled -eq $true
+				} else {
+					$jobs = $Jobs | Where-Object IsEnabled -eq $false
+				}
+			}
```

Comment:
This block of logic needs to be after the `$ExcludeJob` filter, originally after line 75. 

I guess technically it does not matter but just a matter of flow, we filter the `$jobs` down first based on the `-Job` and `-ExcludeJob` objects passed in, then filter on the enabled or disabled.